limesurvey/limesurvey.py

```diff
@@ -187,17 +265,17 @@ def _invoke(self, method: str, *params) -> dict:
         returns:
             The response from the API
         """
+        limesurvey_api_url = getattr(settings, "LIMESURVEY_INTERNAL_API", None)
+        if not limesurvey_api_url:
+            raise Exception("LIMESURVEY_INTERNAL_API not set")
+
         payload = {
             "method": method,
             "params": [self.access_key, *params],
             "id": 1,
         }
 
-        response = requests.post(
-            settings.LIMESURVEY_INTERNAL_API,
-            json=payload,
-            timeout=1,
-        )
+        response = requests.post(url=limesurvey_api_url, json=payload, timeout=1)
```

Comment:
@BryanttV can we increase the timeout? 1 second can be a low timeframe for a proper response if the service is running on low resources